content/en/docs/setup/upgrade/canary/index.md

```diff
@@ -111,6 +111,37 @@ istiod-canary-6956db645c-vwhsk
 
 The output confirms that the pod is using `istiod-canary` revision of the control plane.
 
+## Stable revision labels (experimental)
+
+Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone. Revision tags offer a solution to this. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are stable identifiers that point to revisions and can be used to avoid relabeling namespaces, so rather than relabeling the namespace, an operator can simply change where the namespace's revision tag points.
```

Comment:
Suggestion:

```
Manually relabeling namespaces when moving them to a new revision can be tedious and error-prone. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are a solution to this. [Revision tags](/docs/reference/commands/istioctl/#istioctl-experimental-tag) are stable identifiers that point to revisions and can be used to avoid relabeling namespaces. Rather than relabeling the namespace, a mesh operator can simply change the tag to point to a new revision. All namespaces with that tag will be updated at the same time.
```